tests/components/evohome/const.py

```diff
@@ -0,0 +1,501 @@
+"""The tests for evohome."""
+
+from typing import Final
+
+TEST_USER_ACCOUNT: Final = {
+    "userId": "2512649",
+    "username": "user_2512649@gmail.com",
+    "firstname": "John",
+    "lastname": "Smith",
+    "streetAddress": "1 Main Street",
+    "city": "London",
+    "postcode": "E1 1AA",
+    "country": "UnitedKingdom",
+    "language": "enGB",
+}
+
+TEST_INSTALL_INFO: Final = [
```

Comment:
We usually put large test data in a json fixture file and load that in the test setup phase, eg in a pytest fixture function.

https://github.com/home-assistant/core/blob/59637d23913b85ed005befcc15204c0bec202323/tests/common.py#L575-L587